### PR TITLE
Add gruvbox-light theme

### PR DESCRIPTION
**Pull request by schnerring**
1. Added Gruvbox color palette names as comments to the corresponding colors in the existing `gruvbox-dark` theme
2. Fixed the selection colors that were not yet Gruvbox-themed in the existing `gruvbox-dark` theme
3. Added `gruvbox-light` theme which is basically a copy of `gruvbox-dark` with color values re-mapped to the *light* version of the Gruvbox color scheme
4. Added screenshot and updated readme, so `npm test` succeeds

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,6 +128,9 @@ Thank you so much for contributing!!
 * [__Gruvbox Dark__](themes/prism-gruvbox-dark.css) (by [Azat S.](https://github.com/azat-io))<br />
 [![Gruvbox Dark](screenshots/prism-gruvbox-dark.png)](themes/prism-gruvbox-dark.css)
 
+* [__Gruvbox Light__](themes/prism-gruvbox-light.css) (by [Michael Schnerring](https://schnerring.net))<br />
```
Comment (RunDevelopment) [inline README.md:131]: Could you please link your GitHub instead?

Linking your website in the `@author` tag is fine.

**Review by RunDevelopment — CHANGES_REQUESTED** [review]
Thank you for the PR @schnerring!

One small nit.

File: README.md (hunk 1 above)
Comment (schnerring) [reply README.md]: Done!

**Review by RunDevelopment — APPROVED** [review]

**Comment by RunDevelopment** [comment]
Thank you for contributing @schnerring!

**Comment by schnerring** [comment]
Thanks for the quick merge. Can I expect a release for `npm` soon? 😊

**Comment by RunDevelopment** [comment]
Asking the hard question, huh? ;)

With this PR, we added two new themes since v1.7.0, so I think that warrants a new version. I'll try to get it published by tomorrow.

**Comment by schnerring** [comment]
Lovely stuff, thanks!

**Pull request merged**